flake.nix

```diff
@@ -172,6 +172,13 @@
           # A accessible through 'nix eval --json .#exports'
           nixosModules.homestakeros = {
             imports = [ ./modules/homestakeros ];
+            config = {
+              nixpkgs.overlays = [
+                ethereum-nix.overlays.default
+                outputs.overlays.additions
+                outputs.overlays.modifications
+              ];
+            };
```

Comment:
Can all mkIf true options be put here?

Reply:
Not a bad suggestion but I also like the idea of keeping the configurations that use `options.nix` out of flake.

Reply:
For the record, we had a private conversation about this, and we agreed to add all Nix expressions here that do not use module options. Therefore, there is nothing else to add here, as most of that kind of stuff lives in `system/default.nix`.